modules/nf-core/sourmash/gather/main.nf

```diff
@@ -0,0 +1,44 @@
+process SOURMASH_GATHER {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::sourmash=4.5.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/sourmash:4.5.0--hdfd78af_0':
+        'quay.io/biocontainers/sourmash:4.5.0--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(signature)
+    path(database)
+
+    output:
+    tuple val(meta), path('*.csv')           , optional:true, emit: result
+    tuple val(meta), path('*_unassigned.sig'), optional:true, emit: unassigned
+    tuple val(meta), path('*_matches.sig')   , optional:true, emit: matches
```

Comment:
Missing also outputs for `--save-prefetch` [^1].

[^1] https://nf-co.re/docs/contributing/modules#inputoutput-options point 1

Reply:
Done.